### PR TITLE
Add QUARTZ and ibBTC as approved collateral types

### PR DESCRIPTION
**Pull request by mhairimca**

### CODE REVIEW

File: Add QUARTZ as approved collateral type [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  
+
+This store fee was chosen as it is approximately equivalent to $400 in line with other collateral currencies as determined by  querying CoinGecko on <insert date> 
+  
+
+## Implementation  
+  
+
+This change has no implementations other than the aforementioned governor transactions  
+
```
Comment (therealhexi) [inline Add QUARTZ as approved collateral type:49]: I feel like we could add somewhere about the fact that the final fee for QUARTZ is derived from BSC, and link the BSC Scan contract address https://bscscan.com/token/0xe03fc45dca9c53a178886c77d6914146b51728a5

**Review by therealhexi — APPROVED** [review]
Looks good to me

**Comment by smb2796** [comment]
Can you make this a markdown file and fix the formatting please?

File: Add QUARTZ as approved collateral type.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+## Headers
+| UMIP-xx    |                                     |
+|------------|-------------------------------------|
+| UMIP Title | Add  QUARTZ as collateral currency    |
```
Comment (Reinis-FRP) [inline Add QUARTZ as approved collateral type.md:4]:
```suggestion
| UMIP Title | Add  QUARTZ and ibBTC as collateral currency    |
```

File: Add QUARTZ as approved collateral type.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+- The ibBTC token address 0xc4e15973e6ff2a35cc804c2cf9d2a1b817a8b40f ([https://etherscan.io/token/0xc4e15973e6ff2a35cc804c2cf9d2a1b817a8b40f](https://etherscan.io/token/0xc4e15973e6ff2a35cc804c2cf9d2a1b817a8b40f) needs to be added to the collateral currency whitelist introduced in UMIP-8.  
+
+-  A final fee of 0.0067 ibBTC needs to be added for ibBTC in the store contract
+  
+
+## Rationale  
+  
+
+This store fee for each token was chosen as it is approximately equivalent to $400 in line with other collateral currencies as determined by  querying CoinGecko on <insert date>
```
Comment (Reinis-FRP) [inline Add QUARTZ as approved collateral type.md:42]:
```suggestion
This store fee for each token was chosen as it is approximately equivalent to $400 in line with other collateral currencies as determined by  querying CoinGecko. 
```

**Review by Reinis-FRP — APPROVED** [review]

**Pull request merged**